src/normalize/entities.ts

```diff
@@ -186,8 +186,14 @@ const ExistingAttachmentsInfo = new Entity({
 	part: 'part'
 });
 
+
+const DocumentInfo = new Entity({
+	id : 'id'
```

Comment:
if key and value are same then it should not belong this file
this file should only be used in case of mapping for short keys to it's longer versions

Reply:
Done